### ISSUE TITLE
Query API - Start and end epoch date can't be the same.

### ISSUE THREAD

**Issue by johnwhumphreys** [OpenTSDB/opentsdb#1119]
What if I want to request exactly one data point?  It always makes me request at least 2.

**Comment by manolama** [comment]
Yeah we don't have a way to do that. You can just set the end time to start time + 1 or use 2.4 and the `first` aggregator.
